src/controllers/genre.js

```diff
@@ -1,14 +1,16 @@
 const { Genre } = require('../models/genre');
 
+const log = require('../log');
+
 const getGenres = async (ctx) => {
   try {
     const genres = await Genre.find();
 
     ctx.status = 200;
     ctx.body = genres;
   } catch (error) {
-    ctx.status = 500;
-    ctx.body = error.message;
+    log.error(error, 'Error during getting genres');
+    throw error;
```

Comment:
By throwing error like this your app will crash (you will see errors only in node console). You should return error to users at response. So your previous implementation was all right (you only need to log error and thats all).

Reply:
I decided to check it, but when I write ```throw new Error()```, the user still gets an error message and the error code gets to the console. So maybe writing ```ctx.status = 500;
    ctx.body = error.message;``` is redundant, because Koa has default error handler and application won't stop. Do you agree?

Reply:
But in browser, do you receive error on the page in json format?

Reply:
Oh, you are right. Fix it by adding error logger.